### PR TITLE
About us page image when added will now upload

### PR DESCRIPTION
**Pull request by frimpongopoku**
About us page image addition and deletion now works.

_Might have to makemigrations and migrate_

### CODE REVIEW

File: src/api/store/page_settings.py [hunk 1]
```diff
@@ -1,86 +1,89 @@
...
-      media = Media.objects.create(name="Page-Image", file=image)
-      page_setting.image = media
-      page_setting.save()
-
+      if not image == "None":
+        media = Media.objects.create(name="Page-Image", file=image)
+        page_setting.image = media
+        page_setting.save()
+      else:
+        page_setting.image.delete()
```
Comment (Opoku-Agyemang) [inline src/api/store/page_settings.py:69]: why are we deleting the image?

File: src/api/store/page_settings.py [hunk 2]
```diff
@@ -1,86 +1,89 @@
 from database.models import UserProfile, Community
```
Comment (Opoku-Agyemang) [inline src/api/store/page_settings.py:1]: just add Media at the end here and not reimport

**Review by Opoku-Agyemang — CHANGES_REQUESTED** [review]

File: src/api/store/page_settings.py (hunk 1 above)
Comment (frimpongopoku) [reply src/api/store/page_settings.py:69]: So we can update to remove image when weve already updated

**Comment by BradHN1** [comment]
I will finish testing this and the related PRs this afternoon.

File: src/database/models.py [hunk 3]
```diff
@@ -3,17 +3,17 @@
 from database.utils.constants import *
 from datetime import date, datetime
 from django.utils import timezone
-from .utils.common import  json_loader,get_json_if_not_none, get_summary_info
+from .utils.common import json_loader, get_json_if_not_none, get_summary_info
```
Comment (BradHN1) [inline src/database/models.py:6]: Please don't commit files like this with hundreds of formatting changes.  This is very difficult for me to figure out what was changed.  If there is some automated code formatter you're using, please disable it.

File: src/database/models.py [hunk 4]
```diff
@@ -2308,30 +2288,34 @@ class AboutUsPageSettings(models.Model):
   community = models.ForeignKey(Community, on_delete=models.CASCADE, db_index=True)
   title = models.CharField(max_length=LONG_STR_LEN, blank=True)
   sub_title = models.CharField(max_length=LONG_STR_LEN, blank=True)
-  description = models.TextField(max_length=LONG_STR_LEN, blank = True)
-  images = models.ManyToManyField(Media, blank=True)
-  featured_video_link = models.CharField(max_length=SHORT_STR_LEN, blank = True)
+  description = models.TextField(max_length=LONG_STR_LEN, blank=True)
+  # images = models.ManyToManyField(Media, blank=True)
+  image = models.ForeignKey(Media, blank=True, null=True, on_delete=models.SET_NULL)
```
Comment (BradHN1) [inline src/database/models.py:2293]: All the pageSettings models have possibilities for multiple images, which I would like to preserve even if not used currently.  Don't want this page settings to be different from all the others

**Comment by BradHN1** [comment]
Model changes would break the other pageSettings models.  Superceded by https://github.com/massenergize/api/pull/294

**Pull request closed without merging**